channels/pkg/api/channel.go

```diff
@@ -17,15 +17,14 @@ limitations under the License.
 package api
 
 import (
-	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 type Addons struct {
 	metav1.TypeMeta `json:",inline"`
 
 	// We use v1.ObjectMeta so we don't serialize everything, even though we are using the full api machinery here (yet!)
```

Comment:
Comment needs to be updated.

Reply:
@rdrgmnzs Ahh yeah good eye, if @justinsb doesn't have time to fix that when he fixes the conflict below, would you mind fixing in a follow up?